microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -212,14 +416,13 @@ This experiment replaces the **What's New** tool with the new **Welcome** tool.
 
 The **Welcome** tool opens automatically after each update to Microsoft Edge.  To prevent the display of the **Welcome** tool after each update, clear the checkbox next to **Open tab after each update** under the **Welcome** tool title.
 
-If you prefer the original **What's New** tool, navigate to [Settings](../customize/index.md#settings) > **Experiments** and remove the checkbox next to **Enable Welcome tab**.
+If you prefer the original **What's New** tool, go to [Settings](../customize/index.md#settings) > **Experiments** and then clear the **Enable Welcome tab** checkbox.
 
 :::image type="content" source="../media/experiments-welcome.msft.png" alt-text="Welcome tool" lightbox="../media/experiments-welcome.msft.png":::
```

Comment:
I think this no longer exists since it's no longer an experimental feature.

Reply:
Removed line.